### PR TITLE
feat(collection): add stories

### PR DESCRIPTION
**Pull request by YeSuX**

### CODE REVIEW

File: .storybook/style.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@tailwind base;
+/* @tailwind base; */
```
Comment (productdevbook) [inline .storybook/style.css:1]: why do we do this ?

File: packages/components/collection/src/stories/CollectionDemo.vue [hunk 2]
```diff
@@ -1,59 +1,125 @@
 <!-- eslint-disable no-console -->
```
Comment (productdevbook) [inline packages/components/collection/src/stories/CollectionDemo.vue:1]:
```suggestion
```

File: .storybook/style.css (hunk 1 above)
Comment (YeSuX) [reply .storybook/style.css:1]: image
image

html default style can be reseted by taiwind base.

Comment (YeSuX) [reply .storybook/style.css:1]: So I delete the taiwind base

Comment (productdevbook) [reply .storybook/style.css:1]: i will take a look at this

**Pull request merged**